build/build_tray_icon.ts

```diff
@@ -25,14 +25,14 @@ import fs from "fs-extra";
 
 export async function generateTrayIcon(
   {
-    outputFilename = "tray_icon", // e.g. output tray_icon_dark@2x.png
+    outputFilename = "trayIcon",
```

Comment:
Why were these changed?

Reply:
I've found the issue https://github.com/lensapp/lens/pull/1595 and I assumed that it would be useful to have `trayIconTempltate` file name after generating the icon. If I'm not mistaken for all OS (except mac os) we are using light icon for the tray. @aleksfront am I right? (sorry, I made a mistake and add the dark icon instead of light, I'll fix it).